timbermill-local/src/main/java/com/datorama/oss/timbermill/ElasticsearchClient.java

```diff
@@ -241,16 +250,28 @@ public Task getTaskById(String taskId){
 		return getSingleTaskByIds(boolQueryBuilder, null, "Fetch previously indexed parent tasks", ElasticsearchClient.PARENT_FIELD_TO_FETCH, EMPTY_ARRAY);
 	}
 
-	public Map<String, Task> getLatestOrphanIndexed(ZonedDateTime from) {
+	public Map<String, Task> getLatestOrphanIndexed(ZonedDateTime orphansStartingFrom, ZonedDateTime excludePartialTasksStartingFrom) {
```

Comment:
Let's use only one var for both, meaning check for partials non orphans from the last x mins and partials orphans for beyond the x mins

Reply:
but it means we will bring all partials from env all of the time... are we sure?